### PR TITLE
Replaced all user related stuff with new user models

### PR DESCRIPTION
**Pull request by MyIgel**

### CODE REVIEW

File: includes/pages/admin_active.php [hunk 1]
```diff
@@ -179,18 +210,21 @@ function admin_active()
                 continue;
             }
         }
-        $usr['nick'] = User_Nick_render($usr);
-        $usr['shirt_size'] = $tshirt_sizes[$usr['Size']];
-        $usr['work_time'] = round($usr['shift_length'] / 60)
+
+        $userData = [];
+        $userData['nick'] = User_Nick_render($usr);
+        $userData['shirt_size'] = $tshirt_sizes[$usr->personalData->shirt_size];
```
Comment (msquare) [inline includes/pages/admin_active.php]: This fails when shirt_size is empty.
Add '' => '-' as shirt size to config.default.php.

File: includes/pages/admin_import.php [hunk 2]
```diff
@@ -13,7 +13,8 @@ function admin_import_title()
  */
 function admin_import()
 {
-    global $rooms_import, $user;
+    global $rooms_import;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/admin_import.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/admin_news.php [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+use Engelsystem\Models\User\User;
 
 /**
  * @return string
  */
 function admin_news()
 {
-    global $user, $privileges;
+    global $privileges;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/admin_news.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/admin_questions.php [hunk 4]
```diff
@@ -39,14 +40,14 @@ function admin_new_questions()
  */
 function admin_questions()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/admin_questions.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/admin_user.php [hunk 5]
```diff
@@ -15,7 +16,8 @@ function admin_user_title()
  */
 function admin_user()
 {
-    global $user, $privileges;
+    global $privileges;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/admin_user.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_messages.php [hunk 6]
```diff
@@ -15,12 +16,12 @@ function messages_title()
  */
 function user_unread_messages()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_messages.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_news.php [hunk 7]
```diff
@@ -132,8 +131,7 @@ function display_news($news)
  */
 function user_news_comments()
 {
-    global $user;
-
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_news.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_news.php [hunk 8]
```diff
@@ -200,7 +198,8 @@ function user_news_comments()
  */
 function user_news()
 {
-    global $privileges, $user;
+    global $privileges;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_news.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_questions.php [hunk 9]
```diff
@@ -15,21 +16,21 @@ function questions_title()
  */
 function user_questions()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_questions.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_shifts.php [hunk 10]
```diff
@@ -129,7 +128,7 @@ function load_days()
  */
 function load_types()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_shifts.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/user_shifts.php [hunk 11]
```diff
@@ -284,13 +284,13 @@ function view_user_shifts()
  */
 function ical_hint()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/pages/user_shifts.php]:
```suggestion
    $user = auth()->user();
```

File: includes/sys_menu.php [hunk 12]
```diff
@@ -20,11 +20,11 @@ function page_link_to($page = '', $parameters = [])
  */
 function header_render_hints()
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/sys_menu.php]:
```suggestion
    $user = auth()->user();
```

File: includes/view/Rooms_view.php [hunk 13]
```diff
@@ -12,10 +12,10 @@
  */
 function Room_view($room, ShiftsFilterRenderer $shiftsFilterRenderer, ShiftCalendarRenderer $shiftCalendarRenderer)
 {
-    global $user;
+    $user = Auth()->user();
```
Comment (msquare) [inline includes/view/Rooms_view.php]:
```suggestion
    $user = auth()->user();
```

File: includes/pages/admin_active.php (hunk 1 above)
Comment (MyIgel) [reply includes/pages/admin_active.php]: I really want keep the config file clean -> any "placeholder" stuff should be handelt inside the view.

I think that something like
```php
$userData['shirt_size'] = (isset($tshirt_sizes[$shirtSize]) ? $tshirt_sizes[$shirtSize] : '-');
```
would be better

**Pull request merged**